ui/js/controllers/bugfiler.js

```diff
@@ -36,6 +36,24 @@ treeherder.controller('BugFilerCtrl', [
             $scope.reftest = reftest;
         }
 
+        $scope.isSummaryBlacklisted = function() {
+            var reason = "";
+            if (search_terms.length === 0) {
+                reason = "Selected failure does not contain any searchable terms.";
+            } else {
+                search_terms.forEach(term => {
```

Comment:
I think this can be replaced with 1-2 lines using https://lodash.com/docs/4.17.4#every